app/config.php

```diff
@@ -37,25 +37,79 @@
     Application::class => \DI\factory(function (ContainerInterface $c) {
         $application = new \PhpSchool\WorkshopManager\Application('PHP School workshop manager', '1.1.0', $c);
         $application->command('install workshopName', InstallWorkshop::class, ['add'])
-            ->setDescription('Install a PHP School workshop.');
+            ->setDescription('Install a PHP School workshop.')
+            ->setHelp(<<<'EOF'
+This command requires a <comment>workshopName</comment> code as argument:
+  <info>workshop-manager <comment>workshopName</comment></info>
```

Comment:
I think this is missing `%command.name%`

Reply:
Yes, it was, my bad.